README.md

```diff
@@ -17,10 +17,11 @@ This project is inspired by the [Google-Wide Profiling: A Continuous Profiling I
 
 ### Quickstart
 
-Build conprof binary from the root of the repo directory (requires [bzr](https://bazaar.canonical.com/) and [dot](https://www.graphviz.org/)):
+Build conprof binary from the root of the repo directory (requires [bzr](https://bazaar.canonical.com/) and [dot](https://www.graphviz.org/) without `$GOPROXY`):
```

Comment:
I don't understand what the without goproxy part means. :)

Reply:
Removing as we set GOPROXY explicitly.